byzcoin/bcadmin/main.go

```diff
@@ -151,6 +153,27 @@ var cmds = cli.Commands{
 		},
 		Action: darcCli,
 	},
+	{
+		Name:    "qrcode",
```

Comment:
Please call the sub-command 'qr', which is more concise.

Reply:
Done, qrcode still usable as an alias